src/mplfinance/_utils.py

```diff
@@ -8,7 +8,8 @@
 import datetime
 
 from matplotlib import colors as mcolors
-from matplotlib.collections import LineCollection, PolyCollection
+from matplotlib.patches import Rectangle
```

Comment:
Looks like you can get rid of `Rectangle` and `PatchCollection`.  Probably at one point you thought you would use them.  Thanks.